### PR TITLE
[BIK-878] Add support for sensor export

### PR DESCRIPTION
**Pull request by hb0**

### CODE REVIEW

File: libs/model/src/main/kotlin/de/cyface/model/ExportOptions.kt [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+        }
+
+        /**
+         * Returns the [DataFormat] from it's `#parameterValue` value.
+         *
+         * @param parameterValue The `String` value of the parameterValue.
+         * @return The `DataFormat` for the parameterValue.
+         */
+        @Suppress("unused") // Part of the API
+        fun valueOfParameterValue(parameterValue: String): DataFormat? {
```
Comment (muthenberg) [inline libs/model/src/main/kotlin/de/cyface/model/ExportOptions.kt:111]: I would just call this method `of(parameterValue: String): DataFormat?`. The call would look like `DataFormat.of("json")`. This avoids having 'value' three times in the signature and 'parameterValue' twice.

File: libs/model/src/main/kotlin/de/cyface/model/ExportOptions.kt [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+        }
+
+        /**
+         * Returns the [DataType] from it's `#parameterValue` value.
+         *
+         * @param parameterValue The `String` value of the parameterValue.
+         * @return The `DataType` for the parameterValue.
+         */
+        @Suppress("unused") // Part of the API
+        fun valueOfParameterValue(parameterValue: String): DataType? {
```
Comment (muthenberg) [inline libs/model/src/main/kotlin/de/cyface/model/ExportOptions.kt:167]: same as above

File: libs/model/src/main/kotlin/de/cyface/model/ExportOptions.kt [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+/**
+ * Supported data types which are used during data collection.
+ *
+ * @author Armin Schnabel
+ * @version 1.0.0
+ * @since 3.3.0
+ * @param parameterValue The parameter value which represents the enum option.
+ */
+enum class DataType(@Suppress("MemberVisibilityCanBePrivate") val parameterValue: String) {
```
Comment (muthenberg) [inline libs/model/src/main/kotlin/de/cyface/model/ExportOptions.kt:125]: Why are you suppressing this warning?

**Review by muthenberg — CHANGES_REQUESTED** [review]
Small change request. Otherwise ok.

File: libs/model/src/main/kotlin/de/cyface/model/ExportOptions.kt (hunk 3 above)
Comment (hb0) [reply libs/model/src/main/kotlin/de/cyface/model/ExportOptions.kt:125]: At that time, I didn't access the parameter in this library, but from outside this library. But now it's used from tests, so I am able to remove it without having to ignore warnings while coding.

**Pull request merged**